frontend/containers/profile-header/component.tsx

```diff
@@ -0,0 +1,121 @@
+import { FC } from 'react';
+
+import { Heart as HeartIcon } from 'react-feather';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import Image from 'next/image';
+
+import { noop } from 'lodash-es/noop';
+
+import WebsiteSocialContact from 'containers/website-social-contact';
+
+import Button from 'components/button';
+import LayoutContainer from 'components/layout-container';
+
+import type { ProfileHeaderProps } from './types';
+
+export const ProfileHeader: FC<ProfileHeaderProps> = ({
+  className,
+  logo,
+  title,
+  subtitle,
+  text,
+  originalLanguage,
+  numNotFunded,
+  numFunded,
+  website,
+  social,
+  contact,
+  onFavoriteClick = () => noop,
+  onContactClick = () => noop,
+}: ProfileHeaderProps) => {
+  const intl = useIntl();
+
+  return (
+    <div className={className}>
+      <div className="pt-24 pb-12 bg-center bg-cover sm:pt-40 md:pt-56 bg-radial-dark-green">
+        <LayoutContainer className="flex justify-between">
+          <div className="flex flex-col items-center w-full gap-6 lg:items-end lg:flex-row lg:w-6/12">
+            <div className="p-4 bg-white w-52 h-52 rounded-2xl">
+              <Image
+                className="mx-auto w-52 h-52"
+                src={logo}
+                alt={`${title} ${intl.formatMessage({ defaultMessage: 'logo', id: 'foSyCj' })}`}
+                layout="responsive"
+                width="100%"
+                height="100%"
+                objectFit="contain"
+              />
+            </div>
+            <div className="mb-4 text-center lg:text-left">
+              <h1 className="font-serif text-3xl text-white">{title}</h1>
+              <p className="mt-2 text-xl text-gray-400">{subtitle}</p>
+            </div>
+          </div>
+        </LayoutContainer>
+      </div>
+      <LayoutContainer className="flex flex-col justify-between mt-8 lg:flex-row">
+        <div className="w-full lg:w-6/12">
+          {originalLanguage && (
+            <span className="block mb-4 text-sm text-gray-400">
+              <FormattedMessage
+                defaultMessage="Note: The content of this page was originally written in"
+                id="cOscDz"
+              />
+              <span> {originalLanguage}</span>
+            </span>
```

Comment:
Same issue here. It'd be better to include the variable in the default message.
```suggestion
            <span className="block mb-4 text-sm text-gray-400">
              <FormattedMessage
                defaultMessage="Note: The content of this page was originally written in {language}"
                values={{
                  // Not looking very great and not reusing the constants from `locales.config.json`. We can probably improve that.
                  language:
                    originalLanguage === 'en'
                      ? intl.formatMessage({ defaultMessage: 'English', id: 'WkrNSk' })
                      : originalLanguage === 'pt'
                      ? intl.formatMessage({ defaultMessage: 'Portuguese', id: 'A4UTjl' })
                      : intl.formatMessage({ defaultMessage: 'Spanish', id: '8WtyrD' }),
                }}
                id="jJwDSW"
              />
            </span>
```